ruleset/rules/0998-aws-security-hub-rules.xml

```diff
@@ -0,0 +1,491 @@
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon EMR block public access setting not enabled.</description>
+    <group>nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99844" level="12">
+    <if_sid>99819</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^ES.2$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Elasticsearch domain is publicly accessible.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.2,pci_dss_1.3.4,pci_dss_1.3.6,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+    <mitre>
+      <id>T1530</id>
+    </mitre>
+  </rule>
+
+  <rule id="99845" level="12">
+    <if_sid>99821</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^IAM.4$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - IAM root user access key exist.</description>
+    <group>pci_dss_2.1,pci_dss_2.2,pci_dss_7.2.1,cis_aws_foundations_benchmark_1.12,cis_aws_foundations_benchmark_1.4,nist_800_53_AC.2,nist_800_53_AC.3,nist_800_53_AC.6,</group>
+  </rule>
+
+  <rule id="99846" level="12">
+    <if_sid>99821</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^IAM.6$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Hardware MFA not enabled for the root user.</description>
+    <group>pci_dss_8.3.1,cis_aws_foundations_benchmark_1.14,cis_aws_foundations_benchmark_1.6,nist_800_53_AC.2,nist_800_53_AC.3,nist_800_53_IA.2,</group>
+    <mitre>
+      <id>T1556.006</id>
+    </mitre>
+  </rule>
+
+  <rule id="99847" level="12">
+    <if_sid>99821</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^IAM.9$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - MFA not enabled for the root user.</description>
+    <group>pci_dss_8.3.1,cis_aws_foundations_benchmark_1.13,cis_aws_foundations_benchmark_1.5,nist_800_53_AC.2,nist_800_53_AC.3,nist_800_53_IA.2,</group>
+    <mitre>
+      <id>T1556.006</id>
+    </mitre>
+  </rule>
+
+  <rule id="99848" level="12">
+    <if_sid>99822</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^KMS.3$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - AWS KMS keys is scheduled for deletion.</description>
```

Comment:
Remove the "is".

Reply:
This has been resolved https://github.com/wazuh/wazuh/pull/23234/commits/1d4bfae8c2701e7cc0cda7b3f8df0965bdb225f8.